R/sensitivity-analysis-results.R

```diff
@@ -57,6 +57,8 @@ SensitivityAnalysisResults <- R6::R6Class("SensitivityAnalysisResults",
     #' @param ... Rest arguments.
     print = function(...) {
       private$printClass()

```

Comment:
I think it is a useful addition to the print method.

Reply:
ok